Add missing meta descriptions (#235)
mongodb/docs-php-library: source/reference/method/MongoDBCollection-getNamespace.txt

MongoDB\\Collection::getNamespace()
===================================

.. meta::

>    :description: Retrieve the namespace of a MongoDB collection using the `getNamespace()` method in PHP.

Reviewer: ```suggestion
   :description: Retrieve the namespace of a collection with the MongoDB PHP Library.
```
